packages/office-ui-fabric-react/src/components/Shimmer/Shimmer.styles.ts

```diff
@@ -53,7 +53,7 @@ export function getStyles(props: IShimmerStyleProps): IShimmerStyles {
         position: 'relative',
         overflow: 'hidden',
         transform: 'translateZ(0)',
-        backgroundColor: shimmerColor || palette.neutralLighter,
+        backgroundColor: shimmerColor || semanticColors.listItemBackgroundHovered,
```

Comment:
what semantic relation has `Shimmer` which is a placeholder with `listItemBackgroundHovered`. Same goes to the gradient bellow. You should either have something placeholder related or think of getting as close as possible of a meaning. Perhaps something in the realm of `disabledBackground/bodyDivider`. Shimmer being a placeholder it’s not actionable and we can think of it as disabled (`disabledBackground`). And `bodyDivider` could be used for the darker shade of the gradient as it divides the gradient in 2 sides. Definitely don’t use slots from `listItem` group.

Reply:
@phkuo do you think placeholders like Shimmer worth introducing new slots or we are ok with what I proposed above?

Reply:
OK I will update to `disabledBackground/bodyDivider`, my initial reasoning for using lists was because the wording `disabledBackground` was a bit misleading to me and shimmer looks like it could possibly be a placeholder for loading lists but it could be content too so your feedback definitely makes sense!